src/litdata/streaming/reader.py

```diff
@@ -439,10 +439,13 @@ def _get_folder_size(path: str) -> int:
 
     """
     size = 0
+    _SUPPORTED_EXTENSIONS = (".bin", ".json", ".parquet", ".npy")
+
     for dirpath, _, filenames in os.walk(str(path)):
         for filename in filenames:
-            with contextlib.suppress(FileNotFoundError):
-                size += os.stat(os.path.join(dirpath, filename)).st_size
+            if filename.endswith(_SUPPORTED_EXTENSIONS):
+                with contextlib.suppress(FileNotFoundError):
+                    size += os.stat(os.path.join(dirpath, filename)).st_size
```

Comment:
The idea is not use os.stat which is expensive and not walk the folder.

Instead, we check only the cache folder, not its parents, list the file and use the config to estimate the size

Reply:
> Instead, we check only the cache folder, not its parents, list the file and use the config to estimate the size

yes, while debugging to understand the sizes, I could see it was considering the other dirs(for eg: `remote_dir`) in the tests